main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "testground"
 	app.Usage = "a platform for testing, benchmarking, and simulating distributed and p2p systems at scale"
-	app.Description = "testground is a platform for testing, benchmarking," +
+	app.Description = "testground is a platform for testing, benchmarking, " +
```

Comment:
On Line 22, we should remove the comment to implement a testground version command, since this accomplishes that. 

Also I would be okay with justing the urfave version

Reply:
For now we can use the built in version functionality in `urfave` because of the verbosity flags I think. Removed the TODO.